yt_dlp/YoutubeDL.py

```diff
@@ -1744,6 +1746,19 @@ def __extract_info(self, url, ie, download, extra_info, process):
         else:
             return ie_result
 
+    def fix_deprecated_fields(self, ie_result):
+        deprecated_multivalue_fields = {
+            'artist': 'artist_list',
+            'composer': 'composer_list',
+            'album_artist': 'album_artist_list',
+            'genre': 'genre_list',
+        }
+        for deprecated_field, new_field in deprecated_multivalue_fields.items():
+            if deprecated_field not in ie_result:
+                continue
+            self.deprecation_warning(f'"{deprecated_field}" field is deprecated. Use "{new_field}" instead')
```

Comment:
Let's not add warning till we've migrated the popular extractors. Otherwise every user just ends up with it
```suggestion
```